src/Query/Conditions/Interface/BetweenConditionInterface.php

```diff
@@ -4,15 +4,14 @@
 
 namespace Yiisoft\Db\Query\Conditions\Interface;
 
-use Yiisoft\Db\Expression\ExpressionInterface;
+use Yiisoft\Db\Expression\Expression;
```

Comment:
are sure not ExpressionInterface?

Reply:
The interface is only used to check that it is used but it does not have any methods, because the conditions have different methods, for example only Expression:class has to_string().